apps/openmw/mwgui/windowmanagerimp.cpp

```diff
@@ -1919,16 +1919,12 @@ namespace MWGui
             mInventoryWindow->cycle(next);
     }
 
-    void WindowManager::playSound(const std::string& soundId, bool preventOverlapping, float volume, float pitch)
+    void WindowManager::playSound(const std::string& soundId, float volume, float pitch)
     {
         if (soundId.empty())
             return;
 
-        MWBase::SoundManager *sndmgr = MWBase::Environment::get().getSoundManager();
-        if (preventOverlapping && sndmgr->getSoundPlaying(MWWorld::Ptr(), soundId))
-            return;
-
-        sndmgr->playSound(soundId, volume, pitch, MWSound::Type::Sfx, MWSound::PlayMode::NoEnv);
+        MWBase::Environment::get().getSoundManager()->playSound(soundId, volume, pitch, MWSound::Type::Sfx, MWSound::PlayMode::NoEnv);
```

Comment:
Won't this change now cause the UI sound to get cutoff if it's played again while already playing, rather than the apparently intended behavior of letting the original sound finish without interruption?

Reply:
Should we treat GUI sound differently from other ones?

Reply:
How much work would it be to treat GUI sounds differently?

Reply:
> How much work would it be to treat GUI sounds differently?

Almost none. We just should decide which behaviour is intended:
1. Allow to stack most of GUI sounds, with special case for journal wheel scrolling to prevent overlapping.
It is our current upstream implementation.
2. Play old copy to end and do not launch new copy to prevent overlapping.
3. Take the PR implementation.

Honestly, I can not hear much difference between all three variants (excepts of loudness in first case), so I can not advice which behaviour is better.